db/migrate/20170112175131_migrate_template_to_parameters_macros.rb

```diff
@@ -31,10 +31,13 @@ def up
     end
 
     LookupValue.all.each do |parameter|
-      parameter.value = convert(parameter.value.to_s)
-      if parameter.value_changed?
-        # we need to skip validations so we use #update_attributes
-        parameter.update_attribute :value, parameter.value
+      # just update string values because it would be destructive otherwise
+      if parameter.value.is_a?(String)
```

Comment:
I do not think this will work. The 'value' attribute is serialized as a `String`, so the condition will always be true.

The first two LookupValues have a just string that looks like yaml, the third really has yaml that is serialized into string:
![](https://i.imgur.com/XUKjPdw.png)

viewed directly in db:
![](https://i.imgur.com/pU5qfAI.png)